src/internal/components/pluggableVisualizations/comboChart/tests/PluggableComboChart.spec.tsx

```diff
@@ -331,46 +328,28 @@ describe("PluggableComboChart", () => {
         );
 
         it.each([
-            [
-                AXIS.PRIMARY,
-                COLUMN_AREA_LINE,
-                oneMetricAndCategoryAndStackReferencePoint,
-                [...COMBO_CHART_SUPPORTED_PROPERTIES[AXIS.PRIMARY], ...OPTIONAL_STACKING_PROPERTIES],
-            ],
-            [
-                AXIS.SECONDARY,
-                COLUMN_AREA_LINE,
-                measuresOnSecondaryAxisAndAttributeReferencePoint,
-                [...COMBO_CHART_SUPPORTED_PROPERTIES[AXIS.SECONDARY], ...OPTIONAL_STACKING_PROPERTIES],
-            ],
-            [
-                AXIS.DUAL,
-                COLUMN_AREA_LINE,
-                multipleMetricsAndCategoriesReferencePoint,
-                [...COMBO_CHART_SUPPORTED_PROPERTIES[AXIS.DUAL], ...OPTIONAL_STACKING_PROPERTIES],
-            ],
+            [AXIS.PRIMARY, oneMetricAndCategoryAndStackReferencePoint],
+            [AXIS.SECONDARY, measuresOnSecondaryAxisAndAttributeReferencePoint],
+            [AXIS.DUAL, multipleMetricsAndCategoriesReferencePoint],
         ])(
-            "should update supported properties list when axis is %s and chart type is/are %s",
-            (
-                axis: string,
-                supportedChartTypes: string[],
-                refPoint: IReferencePoint,
-                expectedStackProperties: string[],
-            ) => {
-                supportedChartTypes.forEach(async (chartType: VisualizationObject.VisualizationType) => {
-                    const mockProps = {
-                        ...defaultProps,
-                        pushData: jest.fn(),
-                    };
-
-                    const comboChart = createComponent(mockProps);
-                    refPoint.buckets.forEach((bucket: IBucket) => (bucket.chartType = chartType));
-
-                    const ext = await comboChart.getExtendedReferencePoint(refPoint);
-                    const axisType = get(ext, UICONFIG_AXIS, AXIS.PRIMARY);
-
-                    expect(axisType).toEqual(axis);
-                    expect(comboChart.getSupportedPropertiesList()).toEqual(expectedStackProperties);
+            "should update supported properties list when axis is %s and chart type is column/line/area",
+            (axis: string, refPoint: IReferencePoint) => {
+                const comboChart = createComponent(defaultProps);
+                const refPoints: Promise<IExtendedReferencePoint>[] = COLUMN_AREA_LINE.map(
+                    (chartType: VisualizationObject.VisualizationType) => {
+                        refPoint.buckets.forEach((bucket: IBucket) => (bucket.chartType = chartType));
+                        return comboChart.getExtendedReferencePoint(refPoint);
+                    },
+                );
+
+                Promise.all(refPoints).then(function(extRefPoints: IExtendedReferencePoint[]) {
```

Comment:
Follow this point from [here](https://jestjs.io/docs/en/asynchronous#promises)
`Be sure to return the promise - if you omit this return statement, your test will complete before the promise returned from fetchData resolves and then() has a chance to execute the callback.`
I think we should `return Promise.all(...)`

And please consider using `for of`
For example
```
for (const extRefPoint of extRefPoints) {
...
}
```